azure-pipelines.yml

```diff
@@ -40,49 +28,31 @@ stages:
           vmImage: 'ubuntu-latest'
         steps:
         - template: pipeline/steps/tf-install.yaml
-          parameters:
-            terraformVersion: $(terraformVersion)
         - template: pipeline/steps/tf-init.yaml
           parameters:
-            subscription: $(subscription)
             resourceGroup: $(devopsResourceGroup)
             storageAccount: $(devopsStorageAccount)
-            location: $(location)
-            environment: $(environment)
-            prefix: $(prefix)
-            product: $(product)
         - template: pipeline/steps/tf-plan.yaml
-          parameters:
-            subscription: $(subscription)
-            environment: $(environment)
         - template: pipeline/steps/tf-apply.yaml
+        - template: pipeline/steps/export-keyvault.yaml
           parameters:
-            subscription: $(subscription)
-            environment: $(environment)
+            keyVaultName: hmi-apim-kv-$(environment)
+            secret: hmi-apim-sub-key
 
   - stage: TESTS1
     dependsOn: BUILD
     displayName: Tests - Functional
     jobs:
-      - job:
+      - job: testfunc
         pool:
           vmImage: 'windows-latest'
         steps:
-          # - task: AzureCLI@2
-          #   displayName: Update test file
-          #   inputs:
-          #     azureSubscription: $(subscription)
-          #     scriptType: 'bash'
-          #     scriptLocation: 'inlineScript'
-          #     inlineScript: |
-          #       echo targetSubscriptionKey : \${TEST_SUBSCRIPTION_KEY:$(hmi-apim-svc-sbox-sub-key)} >> application-functional.yaml
-          #     workingDirectory: '$(System.DefaultWorkingDirectory)/src/functionalTest/resources/'
 
           - task: Gradle@2
             inputs:
               gradleWrapperFile: 'gradlew'
               workingDirectory: '$(System.DefaultWorkingDirectory)'
-              tasks: clean functional -DTEST_SUBSCRIPTION_KEY=$(hmi-apim-svc-sbox-sub-key) -DTEST_URL=https://hmi-apim-svc-sbox.azure-api.net -DTEST_HOST=hmi-apim-svc-sbox.azure-api.net
+              tasks: clean functional -DTEST_SUBSCRIPTION_KEY=$(hmi-apim-sub-key) -DTEST_URL=https://hmi-apim-svc-sbox.azure-api.net -DTEST_HOST=hmi-apim-svc-sbox.azure-api.net
```

Comment:
test not sbox?

Reply:
@bugbare-reform - This is not my implementation or file...It came up in the rebase from master....Not part of this implementation.